yarn.lock

```diff
@@ -4423,6 +4443,20 @@ __metadata:
   languageName: node
   linkType: hard
 
+"commander@npm:^8.3.0":
```

Comment:
I'm guessing this is coming from Showdown?

Reply:
There's quite a few dependencies that require commander, including `@hypothesis/frontend-build`.

`8.3.0` specifically is required by `katex`:

```
$ yarn why commander
├─ @hypothesis/frontend-build@npm:4.0.0
│  └─ commander@npm:13.1.0 (via npm:^13.0.0)
│
├─ @hypothesis/frontend-build@npm:4.0.0 [4f7fa]
│  └─ commander@npm:13.1.0 (via npm:^13.0.0)
│
├─ katex@npm:0.16.22
│  └─ commander@npm:8.3.0 (via npm:^8.3.0)
│
├─ nearley@npm:2.20.1
│  └─ commander@npm:2.20.3 (via npm:^2.19.0)
│
├─ showdown@npm:2.1.0
│  └─ commander@npm:9.5.0 (via npm:^9.0.0)
│
├─ sucrase@npm:3.35.0
│  └─ commander@npm:4.1.1 (via npm:^4.0.0)
│
├─ svgo@npm:2.7.0
│  └─ commander@npm:7.2.0 (via npm:^7.2.0)
│
└─ terser@npm:5.19.2
   └─ commander@npm:2.20.3 (via npm:^2.20.0)
```

Reply:
I didn't know the KaTeX package had a CLI. Now I do.